notebooks/official/workbench/spark/spark_sample_notebook.ipynb

```diff
@@ -0,0 +1,1200 @@
+{
```

Comment:
<p>Is there any way to avoid this step? Can it be done before the previous step? Or is this a "chicken or the egg" type problem?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/720/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>It can be avoided only if the user already has a Dataproc cluster and select 'Python 3 on CLUSTER_NAME' when they open this notebook. We can't expect the user already has a cluster so I think it is a necessary process.</p>